lib/screens/config/v2/ol_crowding.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Screens.Config.V2.OLCrowding do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          station: CurrentStopName.t()
+        }
+
+  @enforce_keys [:station]
+  defstruct station: []
```

Comment:
Should the default be `nil` since `CurrentStopName` is just a string?

Reply:
```suggestion
  defstruct station: nil
```